src/librustc_interface/passes.rs

```diff
@@ -711,7 +721,7 @@ impl<'tcx> QueryContext<'tcx> {
     }
 
     pub fn print_stats(&mut self) {
-        self.enter(ty::query::print_stats)
+        self.enter(|tcx| ty::query::print_stats(tcx))
```

Comment:
Why is this changed?

Reply:
It should not be changes. Must have been a merge issue. Thanks for the catch.